app/Resources/views/blog/_comment_form.html.twig

```diff
@@ -20,6 +20,7 @@
             {{ form_label(form.content, 'label.content', { label_attr: { class: 'hidden' }}) }}
 
             {% if not form.vars.valid %}
+                {{ form_errors(form) }}
                 {{ form_errors(form.content) }}
```

Comment:
@voronkovich  I don't think `{{ form_errors(form.content) }}` is redundant, since `form_errors(form)` renders any "global" errors only, whereas `{{ form_errors(form.content) }}` renders the field error (e.g. when entering a comment with too few characters).
I think you are right about the `if` statement. I'm going to update the PR.